keys/ranges.go

```diff
@@ -0,0 +1,45 @@
+// Copyright 2015 The Cockroach Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+// implied. See the License for the specific language governing
+// permissions and limitations under the License. See the AUTHORS file
+// for names of contributors.
+//
+// Author: Marc Berhault (marc@cockroachlabs.com)
+
+package keys
+
+import "github.com/cockroachdb/cockroach/proto"
+
+// Range describes a kv range.
+type Range struct {
```

Comment:
We don't use "span" anywhere else, but we do use "range" all over with a
very specific meaning. When I first read your PR description about
introducing ranges I thought it was going to be about doing some additional
splits in the bootstrap process.
On Aug 27, 2015 11:19 PM, "marc" <notifications@github.com> wrote:

> In keys/ranges.go
> <https://github.com/cockroachdb/cockroach/pull/2281#discussion_r38147447>:
>
> > +//
> > +// Unless required by applicable law or agreed to in writing, software
> > +// distributed under the License is distributed on an "AS IS" BASIS,
> > +// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
> > +// implied. See the License for the specific language governing
> > +// permissions and limitations under the License. See the AUTHORS file
> > +// for names of contributors.
> > +//
> > +// Author: Marc Berhault (marc@cockroachlabs.com)
> > +
> > +package keys
> > +
> > +import "github.com/cockroachdb/cockroach/proto"
> > +
> > +// Range describes a kv range.
> > +type Range struct {
>
> Is this terminology we use anywhere else? If not, I'm hesitant to just
> introduce it in this one place.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cockroachdb/cockroach/pull/2281/files#r38147447>.
>